src/GraphQL/Types/Composite/AutoRegisteringHelper.cs

```diff
@@ -10,13 +10,36 @@ namespace GraphQL.Types
 {
     internal static class AutoRegisteringHelper
     {
+        internal static void ConfigureGraphType<TSourceType>(IGraphType graphType)
+        {
+            var classType = typeof(TSourceType);
+
+            // Apply [Description] attribute
+            var descriptionAttribute = classType.GetCustomAttribute<DescriptionAttribute>();
```

Comment:
From System.ComponentModel?

Reply:
Yes, basically the same as `propertyInfo.Description()` does.  I will probably update this to use the same extension method or something.  Not sure yet.